cg/meta/archive/ddn_dataflow.py

```diff
@@ -222,6 +229,14 @@ def post_request(self, url: str, headers: dict) -> GetJobStatusResponse:
         Raises:
              HTTPError if the response code is not ok.
         """
+
+        LOG.info(
+            "Sending request with headers: \n"
+            + f"{headers} \n"
```

Comment:
It will contain the JWT token, but not any usernames or passwords. Mainly added it for debugging purposes, though. Is it good practice to not log tokens?

Reply:
@seallard Probably not useful when we are merging this, it was mainly to see that any token at all was set